templates/admin.html

```diff
@@ -43,6 +48,14 @@
     for (let f of document.getElementsByName("del-confirm")) {
 	f.onsubmit = function() { return confirm('{{_("Sind Sie sicher?")}}'); };
     }
+    
+    b = document.getElementById("obfuscateUsername");
+    if (b != null) {
```

Comment:
Is that really an obfuscation or just a random pseudonym? If the latter, let's just call it that way.

Also this probably should be configurable and switched off by default.